repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -91,13 +90,13 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, chain: FilterChain): Unit = {
     /**
-     * STATIC REFERENCE TO CONFIG
-     */
+      * STATIC REFERENCE TO CONFIG
```

Comment:
http://docs.scala-lang.org/style/scaladoc.html

Apparently it's the Scaladoc style.